Scripts/RemediationScripts/Remediate-CdnFrontDoorMinTLSVersion.ps1

```diff
@@ -903,109 +701,113 @@ function Reset-FrontDoorRequiredTLSVersion
-                $secretResoure = New-AzFrontDoorCdnResourceReferenceObject -Id $secretObj.Id
-                $updateTlsSetting = New-AzFrontDoorCdnCustomDomainTlsSettingParametersObject -CertificateType $CertificateType -MinimumTlsVersion $prevMinimumTlsVersion -Secret $secretResoure
-                $resource = Update-AzFrontDoorCdnCustomDomain -ResourceGroupName $resourceGroupName -ProfileName $resourceName -CustomDomainName $DomainName -TlsSetting $updateTlsSetting
-                    
-                if($null -ne $resource -and $updateTlsSetting.MinimumTlsVersion -eq $prevMinimumTlsVersion )
-                {
-                    $frontdoorEndpoint.isMinTLSVersionRolledBackCustomDomainPostRemediation = $true
-                    $frontdoorEndpoint.isMinTlsVersionSetOnCustomDomain = $false
-                    $frontdoorEndpoint.minimumTlsVersion = $requiredMinTLSVersion
-                    $frontDoorEndpointsRolledBack += $frontdoorEndpoint
-                    Write-Host "Minimum required TLS version for Front Door has been Rolled back successfully for FrontDoor" ($resourceName) "with Domain" ($DomainName) -ForegroundColor $([Constants]::MessageType.Update)
-                    Write-Host $([Constants]::SingleDashLine)
+                 
                 }
                 else
                 {
-                    $frontDoorsEPSkipped += $frontdoorEndpoint
+                    $frontDoorsDomainsSkipped += $frontdoorDomain
                     $logResource = @{}
                     $logResource.Add("DomainName",($_.DomainName))
                     $logResource.Add("ResourceGroupName",($_.ResourceGroupName))
-                    $logResource.Add("ResourceName",($_.ResourceName))
-                    $logResource.Add("Reason", "Error while Rolling back the minimum required TLS version. Skipping this Front Door.")
+                    $logResource.Add("ResourceName",($_.ProfileName))
+                    $logResource.Add("Reason", "Error while Rolling back the minimum required TLS version. Skipping this Front Door CDN Profiles Domain.")
                     $logSkippedResources += $logResource
-                    Write-Host "Error while Rolling back the minimum required TLS version for this Front Door." -ForegroundColor $([Constants]::MessageType.Error)
+                    Write-Host "Error while Rolling back the minimum required TLS version for this Front Door CDN Profiles Domain." -ForegroundColor $([Constants]::MessageType.Error)
                     Write-Host "Skipping this Front Door." -ForegroundColor $([Constants]::MessageType.Warning)
                     Write-Host $([Constants]::SingleDashLine)
                     return
-                }
-                    
+                }                    
             }
             catch
             {
-                $frontDoorsEPSkipped += $frontdoorEndpoint
+                $frontDoorsDomainsSkipped += $frontdoorDomain
                 $logResource = @{}
```

Comment:
why autoremediation code in rollback function anyway? 

Remove all the autoremediation code blocks